components/common/textLink.js

```diff
@@ -1,13 +1,24 @@
-import { Link } from '@chakra-ui/core'
+import { Link as ChakraLink } from '@chakra-ui/core'
+import Link from 'next/link'
 import PropTypes from 'prop-types'
 
 import styles from './textLink.module.scss'
 
 // destructuring children so if someone passes children (which we don't want), it doesn't funk things up
-const TextLink = ({ href, text, children, ...props }) => (
-  <Link href={href} className={styles.link} {...props}>
-    {text}
-  </Link>
+const TextLink = ({ href, text, children, mailLink, ...props }) => (
+  <>
+    {!mailLink ? (
+      <Link href={href}>
+        <ChakraLink className={styles.link} {...props}>
+          {text}
+        </ChakraLink>
+      </Link>
+    ) : (
+      <ChakraLink href={href} className={styles.link} {...props}>
+        {text}
+      </ChakraLink>
+    )}
+  </>
```

Comment:
Since we're styling the `TextLink` with an SCSS module, we don't need to accept style props (the style should never change), so I don't think there is need for any of this to be changed.

Reply:
all i did here was add a next Link wrapper to the textLink (unless it's a mailLink). this is because next/link's route around the app like it's a SPA instead of re rendering the app as if it's linked from an external page.

Reply:
going to merge for now to avoid merge conflicts in a PR today! let's shelve this convo